config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Hydra::Application.routes.draw do
 
+  namespace :aeon do
```

Comment:
just checking, do you want this bit included in the PR now?

Reply:
nope, that goes in a separate branch. got bit by letting `git commit -am` become a habit back when git was new and we didn't know any better ;-)
I've reverted the extra change